CHANGELOG.md

```diff
@@ -47,6 +47,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Using `Image`, `ByteImage` from `datumaro.util.image` - these classes
   are moved to `datumaro.components.media`
   (<https://github.com/openvinotoolkit/datumaro/pull/538>)
+- Using `smooth_line` from `datumaro.util.annotation_util` - the function
+  is renamed to `approximate_line` and has updated interface
+  (<https://github.com/openvinotoolkit/datumaro/pull/592>)
```

Comment:
This can't be listed under "Deprecated" anymore, because the function was removed.

Reply:
Fixed